### DISCUSSION TITLE
LAN IP direct connection

### DISCUSSION THREAD

**Discussion by catcherxuefeng** [rustdesk/rustdesk#7945] [Q&A] [answered]
Establishing virtual LAN technology with VPN technology such as Zerotire or TailScale, I hope to use IP to directly connect RustDesk

**Comment by rustdesk** [comment, marked as answer]
https://github.com/rustdesk/rustdesk/wiki/FAQ#i-only-use-rustdesk-for-a-few-devices-on-my-local-network-with-no-internet-connectivity-can-i-still-use-rustdesk-with-direct-ip-access
